nuitka/OptionParsing.py

```diff
@@ -1267,6 +1267,15 @@ def _process_long_opt(self, rargs, values):
     help="Add executable icon for onefile binary to use. Can be given only one time. Defaults to Python icon if available.",
 )
 
+linux_group.add_option(
+    "--linux-onefile-compression",
+    action="store",
+    dest="appimage_compression",
```

Comment:
There is something for choices, that will allow the option parsing to check, `choices=("original", "runtime", "frozen")` is used elsewhere in the file.

Reply:
Thanks for the tip!